lib/ansible/module_utils/network/iosxr/argspec/facts/facts.py

```diff
@@ -0,0 +1,29 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+"""
+The arg spec for the iosxr facts module.
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+class FactsArgs(object):  # pylint: disable=R0903
+    """ The arg spec for the iosxr facts module
+    """
+
+    def __init__(self, **kwargs):
+        pass
+
+    choices = [
+        'all',
+        'lacp'
```

Comment:
@NilashishC it's better to have all resources appended here and in facts/facts.py, otherwise with each merge, u'll have to rebase all other resource PR needed to be merged.

Reply:
Given that there will be a gap between consecutive resource module merges, if we add all the options before hand, and update the facts doc to reflect the available choices, it may be misleading to the user, as in, one can specify a option, which, would technically pass, but would return nothing.

Reply:
@NilashishC I think I confused u by saying all resource module :D, what I meant was resource module from iosxr interface to vlans as it'll be merged in another week max. Also, u can go ahead n merge the PR as it was just a suggestion, I had already approved the PR. :)

Reply:
@justjais thanks! I should have the iosxr_lacp_interfaces merge ready by tomorrow. Will update the facts in that.